### DISCUSSION TITLE
<title>Lost my two factor acoount email and password

### DISCUSSION THREAD

**Discussion by skytheakash29** [community/community#68082] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello! 
I had an main github account created by my domain email that was hosted in.Suddenly my domain and hosting was expired and I lost access to it. After that I lost also my github account password. Now I can't forgot my account password. So how I can get back my that ccount. pardon my mistake.

**Comment by kamlendras** [comment, marked as answer]
Hi,
[Go to ](https://support.github.com/contact/account?contact[type]=account&flow=2fa)
If u had two factor auth then they might can help you,
 If that did't works then get a ticket . ;)
